### ISSUE TITLE
How can i make new sprite font?

### ISSUE THREAD

**Issue by MaksPRO12667** [ShadowMario/FNF-PsychEngine#15785]
### Describe your problem here.

The standard sprite font used is 'Alphabet' but i wanna add another one like this, but with its own sprites (custom sprite font)
I can't find any information about this

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows x64

**Comment by KELOTOMU** [comment]
You need to make one yourself by editing the alphabet spritesheet or just make one yourself but make sure it has all xml names right. The fla for it is in the `art/flashFiles/`

**Comment by MaksPRO12667** [comment]
> You need to make one yourself by editing the alphabet spritesheet or just make one yourself but make sure it has all xml names right. The fla for it is in the `art/flashFiles/`

No, i mean create custom sprite font, so that there would be both an alphabet and a new alphabet
